NodeChrome/Dockerfile

```diff
@@ -41,24 +41,18 @@ USER seluser
 # Latest released version will be used by default
 #============================================
 ARG CHROME_DRIVER_VERSION
-RUN CHROME_STRING=$(google-chrome --version) \ 
```

Comment:
Please make changes only on the `Dockerfile.txt` file.

Reply:
ok, i reverted the change